mscore/file.cpp

```diff
@@ -480,8 +480,11 @@ MasterScore* MuseScore::getNewFile()
       {
       if (!newWizard)
             newWizard = new NewWizard(this);
-      else
+      else {
+            newWizard->removePage(newWizard->Page::Template);
```

Comment:
What happens with the previously created NewWizardPage4 object? Does it still need to be deleted?